include/zendoo_mc.h

```diff
@@ -928,12 +1005,16 @@ extern "C" {
         uint32_t num_constraints,
         const path_char_t* params_dir,
         size_t params_dir_len,
-        CctpErrorCode* ret_code
+        CctpErrorCode* ret_code,
+        bool compress_vk = true,
+        bool compress_pk = true
```

Comment:
Is there any case in which we should compress the verification key and keep the proving key uncompressed (or the other way around)? If not, we might use a unique `compress` flag for both keys.

Reply:
In reality we never want to save the pk uncompressed (cause it's very big). The vk is pretty small so it's doable instead